framework/messages/hi/yii.php

```diff
@@ -0,0 +1,138 @@
+ * You may modify this file by translating the extracted messages.
+ *
+ * Each array element represents the translation (value) of a message (key).
+ * If the value is empty, the message is considered as not translated.
+ * Messages that no longer need translation will have their translations
+ * enclosed between a pair of '@@' marks.
+ *
+ * Message string can be used with plural forms format. Check i18n section
+ * of the guide for details.
+ *
+ * NOTE: this file must be saved in UTF-8 encoding.
+ */
+return [
+    ' and ' => ' और ',
+    '(not set)' => '(स्थापित नहीं)',
+    'An internal server error occurred.' => 'सर्वर की एक आंतरिक खामी हुई।',
+    'Are you sure you want to delete this item?' => 'क्या आप सुनिश्चित रूप से इस चीज को नष्ट करना चाहते हैं?',
+    'Delete' => 'नष्ट',
+    'Error' => 'खामी',
+    'File upload failed.' => 'फ़ाइल अपलोड असफल हुआ।',
+    'Home' => 'घर',
+    'Invalid data received for parameter "{param}".' => 'पैरामीटर "{param}" के लिए अमान्य माहिती मिली।',
+    'Login Required' => 'लॉगिन आवश्यक',
+    'Missing required arguments: {params}' => 'आवश्यक तर्क: {params} अनुपस्थित है',
+    'Missing required parameters: {params}' => 'आवश्यक पैरामीटर: {params} अनुपस्थित है',
+    'No' => 'नहीं',
+    'No results found.' => 'कोई परिणाम नहीं मिला।',
+    'Only files with these MIME types are allowed: {mimeTypes}.' => 'इन MIME प्रकारों वाली केवल फ़ाइलों की अनुमति है: {mimeTypes}।',
+    'Only files with these extensions are allowed: {extensions}.' => 'इन एक्सटेंशन वाली केवल फाइलों की अनुमति है: {extensions}।',
+    'Page not found.' => 'पृष्ठ नहीं मिला।',
+    'Please fix the following errors:' => 'कृपया निम्नलिखित खामीयां सुधारें:',
+    'Please upload a file.' => 'कृपया एक फ़ाइल अपलोड करें।',
+    'Powered by {yii}' => '{Yii} द्वारा संचालित',
+    'Showing <b>{begin, number}-{end, number}</b> of <b>{totalCount, number}</b> {totalCount, plural, one{item} other{items}}.' => '<b>{totalCount, number}</b> {totalCount, plural, one{चीज} other{चीज़े}} में से दिखा रहा है <b>{begin, number}-{end, number}</b>।',
+    'The combination {values} of {attributes} has already been taken.' => '{attributes} की {values} का संयोजन पहले से ही लिया जा चुका है।',
+    'The file "{file}" is not an image.' => 'फ़ाइल "{file}" एक छवि नहीं है।',
+    'The file "{file}" is too big. Its size cannot exceed {formattedLimit}.' => 'फ़ाइल "{file}" बहुत बड़ी है। इसका माप {formattedLimit} से अधिक नहीं हो सकता है।',
+    'The file "{file}" is too small. Its size cannot be smaller than {formattedLimit}.' => 'फ़ाइल "{file}" बहुत छोटी है। इसका माप {formattedLimit} से छोटा नहीं हो सकता है।',
+    'The format of {attribute} is invalid.' => '{attribute} का प्रारूप अमान्य है।',
+    'The image "{file}" is too large. The height cannot be larger than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'छवि "{file}" बहुत बड़ी है। ऊंचाई {limit, number} {limit, plural, one{पिक्सेल} अन्य {पिक्सेलस}} से बड़ी नहीं हो सकती।',
```

Comment:
`{limit, plural, one{पिक्सेल} अन्य {पिक्सेलस}} से बड़ी नहीं हो सकती।'` looks wrong. I think part of the ICU special syntax was translated. See https://www.yiiframework.com/doc/guide/2.0/en/tutorial-i18n#plural

Reply:
Great catch. Fixed by 284f2d9fc569e95c259dcbb101cb80e7d3f67ee1